pyroengine/__init__.py

```diff
@@ -1,6 +1,2 @@
-# Copyright (C) 2020-2022, Pyronear.
```

Comment:
Why do you remove the header here ?

Reply:
files with no added value code (`__init__.py` and `version.py`) have no reason to include the header 
It's just adding commented sections in files that have barely anything